docs/src/stack_and_queue.md

```diff
@@ -6,10 +6,40 @@ type, which respectively provide interfaces for LIFO and FIFO access.
 Usage of Stack:
 
 ```julia
-s = Stack{Int}()
-push!(s, x)
-x = top(s)
-x = pop!(s)
+s = Stack{Int}()            # create a stack
+isempty(s)                  # check whether the stack is empty
+length(s)                   # get the number of elements
+eltype(s)                   # get the type of elements
+push!(s, 1)                 # push back a item
+top(s)                      # get a first item
+pop!(s)                     # get and remove a first item
+empty!(s)                   # make a stack empty
+iterate(s::Stack)           # Get a LIFO iterator of a stack
+reverse_iter(s::Stack{T})   # Get a FILO iterator of a stack
+s1 == s2                    # check whether the two stacks are same
+```
+
+Examples of Stack:
```

Comment:
Maybe we do not need this here and in the docstring for stack itself?

Delete one or the other?
Or are they different?

Reply:
OK. I deleted the doc's one.